src/ngraph/runtime/cpu/builder/leaky_relu.cpp

```diff
@@ -30,21 +30,21 @@ namespace ngraph
         namespace cpu
         {
             template <>
-            void Builder::BUILDER_DECL(ngraph::op::CPULeakyRelu)
+            void Builder::BUILDER_DECL(ngraph::op::LeakyRelu)
```

Comment:
Let's keep the `ngraph::op::LeakyRelu` -> `ngraph::op::CPULeakyRelu` change.

Reply:
Done